app/controllers/tags_controller.rb

```diff
@@ -0,0 +1,21 @@
+class TagsController < ApplicationController
```

Comment:
While making this, I was thinking about taking the `Api::AutocompleteController` out of the `Api` namespace since these controllers are not part of an API for external developers but simply internal controllers returning `JSON` documents.